test/index.test.js

```diff
@@ -252,6 +252,35 @@ test('metrics with labels', done => {
     });
 });
 
+test('new metrics with labels', done => {
+    expect.hasAssertions();
+    const consumer = new PrometheusMetricsConsumer({ client: promClient });
+
+    const source = src([
+        {
+            name: 'my_new_counter_with_labels',
+            description: '.',
+            podlet: 'recommendations',
+            url: 'http://mylayout.com',
+            method: 'GET',
+            status: 200,
```

Comment:
Any resons for the `podlet`, `url`, `method` and `status` fields? Would'nt it be better to test with an instance of the metric class here also?

Reply:
yeah, just a quick fix copy and paste from another test so I can get this fixed up.

Reply:
I'll add the metrics object

Reply:
done